regions_www/tabs/region_stats/school/lsoa/avon/region_stats.html

```diff
@@ -63,7 +65,7 @@ <h3>Cycling levels in each scenario for Avon</h3>
 
 <p>Table 1 summarises the proportion of children cycling, walking, driving and using other modes under each scenario in Avon, based on aggregating across all residents in all the LSOA zones.</p>
 
-<p><em>Table 1: Travel to school mode share in the NSC 2011 and in scenarios among 141,727 children living in Avon</em> </p>
+<p><em>Table 1: Travel to school mode share in the NSC 2011 and in scenarios among 141,726 children living in Avon</em> </p>
```

Comment:
We lost one!

Reply:
oh god above - where did he go?! sigh...having a look... :/

Reply:
ok, so, I am pretty sure it is because I a) tweaked the inclusion criteria for the national dataset of children included in the layer and the nb) re-ran the random section of the code in which the 0.3% children with an unknown home LSOA are probabilistically allocated the home LSOA of another child in the same school using the same mode.  I do use seeds in step b), so it does give the same result each time if the input dataset is the same, but did not in this case because the input dataset had been altered by my upstream tweaking in step a). [that sort of modification should not be needed any more once we have agreed the methods for the schools layer, which will be before the final layer goes live]

The upshot was that one unknown-home-LSOA child previously allocated Avon has now been allocated to Somerset – ditto for Gloucestershire. I'm relieved to have found them!

Reply:
That sounds pretty comprehensive search for the lost child.
Thank you for the explanation, it is a shame the 0.3% cause us (mainly you) such grief but it is good that I know some more of the process.